src/renderer/components/commands-action-button.tsx

```diff
@@ -210,6 +210,7 @@ export class GistActionButton extends React.Component<
         message:
           'Updating Fiddle Gist failed. Are you connected to the Internet and is this your Gist?',
         detail: `GitHub encountered the following error: ${error.message}`,
+        buttons: ['OK'],
```

Comment:
Should this be adopted in other dialog boxes as well (Say, for example <a href = 'https://github.com/electron/fiddle/blob/84831af2b7095e2e967d2294a3d86e92f32b024f/src/renderer/components/commands-action-button.tsx#L209'>this</a>)?

Reply:
This should probably be `Ok` not `OK` - additionally, we can perhaps be more specific:
```js
const isLinux = process.platform === 'linux'
...
...(isLinux && { buttons: ['Ok'] })
``` 

but i'm happy to defer to @erickzhao on this one!

Reply:
Oh, is there any reason to be specific? Since adding `Ok` button in case of `Windows`/`MacOS`, just overrides the default value (and is similar to previous case).